posts/2022-12-14-android-ndk-update.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Updating the Android NDK in Rust <version>"
+author: Android Platform Team
+description: "Modernizing Android support in Rust"
+---
+
+We are pleased to announce that Android platform support in Rust will be
+modernized in Rust <version> as we update the target NDK from r17 to r25.  As a
+consequence the minimum supported API level will increase from 15 (Jelly Bean)
+to 19 (KitKat).
+
+In NDK r23 Android switched to using LLVMs `libunwind` for all architectures.
+This meant that
+1. If a project were to target NDK r23 or newer with previous versions of Rust
+   a workaround would be required to redirect attempts to link against `libgcc`
+   to instead link against `libunwind`.  Following this update this workaround
+   will no longer be necessary.
+2. If a project targets an NDK between r17 and r23 it will need to be updated to
```

Comment:
Technically, I believe a project can target a version as low as `r9b` today. Maybe earlier, but that's the oldest one I recall testing. Might be simplest to say "If a project targets NDK version r22 or lower" instead of specifying a lower bound.

Reply:
Done